go/iot/arduino/protocol.ino

```diff
@@ -116,9 +129,11 @@ void sendData() {
   JsonObject& root = jsonBuffer.createObject();
   JsonObject& measure = root.createNestedObject("Measure");
   measure["Name"] = "my.org/measure/Measure_Test";
-  measure["MeasureType"] = "int64";
-  measure["MeasureValue"] = "9";
+  measure["Measurement"] = "9";
 
+  JsonArray& tagValues = root.createNestedArray("TagKeys");
```

Comment:
This array should be called `tagKeys`. Also, I think it's better to have a mapping from tag keys to values, instead of having two arrays.

Reply:
okay.

Reply:
Despite I think you advice makes sense, I am not sure whether Go encoding package could parse this object. I would have a try. If it fails, I would use two arrays.

Reply:
SGTM